src/js/gi/selectors/calculator.js

```diff
@@ -321,9 +327,12 @@ const getDerivedValues = createSelector(
     }
 
     // getTuitionFeesTerm3
-    if (isOJT || oldGiBill || (giBillChapter === 31 && isFlightOrCorrespondence)) {
-      tuitionFeesTerm3 = 0;
-    } else if (inputs.calendar === 'semesters' || inputs.calendar === 'nontraditional' && numberOfTerms < 3) {
+    const shouldHaveNoTuitionFeesTerm3 =
```

Comment:
I think the thousand commas are really important to the readability of the tool... Does this affect affect those? I just saw a screen shot where they were missing and it was hard to read the values.

https://github.com/department-of-veterans-affairs/vets-website/pull/5107